SmartVision/WindowsForms/WebcamInput.cs

```diff
@@ -24,8 +24,7 @@ public class WebcamInput
         private static readonly BroadcastBlock<byte[]> buffer = new BroadcastBlock<byte[]>(item => item);
         private static VideoCapture capture; // Takes video from camera as image frames
         private static Task taskConsumer;
-        private static HttpClientWrapper httpClientWrapper = new HttpClientWrapper();
-        private static FaceApiCalls faceApiCalls = new FaceApiCalls(httpClientWrapper);
+        private static readonly FaceApiCalls faceApiCalls = new FaceApiCalls(new HttpClientWrapper());
```

Comment:
I think it is better to make it "const", as "readonly" allows to change variable in the constructor. Just to make sure that faceApiCalls never gets changed

Reply:
I disagree about this, as a static readonly cannot be modified in constructor (static constructor is called first). Otherwise, const seems more common with literals, anyway

Reply:
Okay my badass boy :*